src/web/lib/contributions.ts

```diff
@@ -24,6 +24,9 @@ const DAILY_ARTICLE_COUNT_KEY = 'gu.history.dailyArticleCount';
 const WEEKLY_ARTICLE_COUNT_KEY = 'gu.history.weeklyArticleCount';
 export const NO_RR_BANNER_TIMESTAMP_KEY = 'gu.noRRBannerTimestamp'; // timestamp of when we were last told not to show a RR banner
 
+// See https://github.com/guardian/support-dotcom-components/blob/main/module-versions.md
+export const ModulesVersion = 'v1';
```

Comment:
As this is a static variable that never changes, we can follow the convention of the upper snake case
```suggestion
export const MODULES_VERSION = 'v1';
```